etc/launcher-configuration.ini

```diff
@@ -0,0 +1,15 @@
+port = 9011
```

Comment:
this file is redundant

Reply:
True, interesting that it was not in the gitignore file